app/layout.tsx

```diff
@@ -1,28 +1,31 @@
-import { GeistSans } from 'geist/font'
-import './globals.css'
+import { GeistSans } from "geist/font";
+import "./globals.css";
 
 const defaultUrl = process.env.VERCEL_URL
   ? `https://${process.env.VERCEL_URL}`
-  : 'http://localhost:3000'
+  : "http://localhost:3000";
 
 export const metadata = {
   metadataBase: new URL(defaultUrl),
```

Comment:
So this is the configuration you were talking about i guest

Reply:
the configuration for supabase

Reply:
yes and there are other configurations like the env and the utils folder
![image](https://github.com/Gmarvis/whatsapp-clone/assets/106551910/d841c480-060f-4c36-a59b-529f53ff72d0)